other/ethhub-cftc-response.md

```diff
@@ -162,11 +162,23 @@ Ethereum has operated uninterrupted for three years as a decentralized blockchai
 
 ### 16. What impediments or risks exist to the reliable conversion of Ether to legal tender? How do these impediments or risks impact regulatory considerations for Commission registrants with respect to participating in any transactions in Ether, including the ability to obtain or demonstrate possession or control or otherwise hold Ether as collateral or on behalf of customers?
 
-Currently, off-ramps to legal tender from Ether are mostly supported by centralized exchanges such as Coinbase or Gemini. These centralized exchanges present a systemic risk to all cryptocurrencies as they maintain custody of customer's cryptocurrencies. These centralized exchanges become a single point of attack or failure. Some exchanges also currently lack many of the controls traditional financial institutions managing custody are required to implement. Some of these entities are unaudited \(both for financials and IT Controls\), unlicensed, and under-/uninisured. Customers can be at significant risk by relying on these central entities to provide custody not only due to the possibility of hacks, but also due to the inability of customers to provably verify solvency on an ongoing basis.
+Ether can be converted to legal tender in two ways: directly via sale on an exchange or indirectly via a sale to another cryptocurrency and then to an exchange. Regarding a direct sale into fiat, Ether faces the same challenges as any other cryptocurrency: volatile price changes and immature exchanges that offer fiat-to-Ether conversion. Both factors contribute significant risk to the acquisition and holding of cryptocurrencies.
```

Comment:
Does fiat currency need defined? Can we simply say fiat by itself or should it be fiat currency each time?

Reply:
I assume CFTC understands what we mean by fiat, especially in context with discussing legal tender. I think it's one of those things where you optionally say fiat currency once at the top of an answer, but only use "fiat" for the rest of the answer. As for defining it, I don't think that's necessary. The definition of fiat is clear, and if the readers don't know what fiat is, I think there are bigger problems than our lack of definition...